source/slang/slang-name.h

```diff
@@ -39,6 +39,9 @@ String getText(Name* name);
 /// Get the text as unowned string slice 
 UnownedStringSlice getUnownedStringSliceText(Name* name);
 
+// Get a name as a C style string, or nullptr if name is nullptr
+const char* getCstr(Name* name);
```

Comment:
Nitpick: Should this be `CStr`?

Just to double-check: the main reason to have this is that it is robust in the case of a null `Name*`?